projects/ngx-formentry/src/components/ngx-datetime-picker/ngx-datetime-picker.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit, Input, forwardRef } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import * as moment_ from 'moment';
+const moment = moment_;
+
+@Component({
+  selector: 'ngx-datetimepicker',
+  templateUrl: './ngx-datetime-picker.html',
+  styleUrls: ['./ngx-datetime-picker.css'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => NgxDatetimeComponent),
+      multi: true
+    }
+  ]
+})
+export class NgxDatetimeComponent implements OnInit, ControlValueAccessor {
+  value: String | Date = '';
+  isDisabled: boolean;
+
+  onChange = (_: any) => { }
+  onTouch = () => { }
+
+  constructor() { }
+
+  ngOnInit() {
+  }
```

Comment:
Is this intentional or should we consider removing empty constructor and init functions?